### PR TITLE
Add link to attendance sheet in playbook/activities.md

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

**Comment by maxwofford** [comment]
This closes #207

@zachlatta Could I get eyes on this?

**Comment by zachlatta** [comment]
What if we created a "Hackers arrive" section in `activities.md` that mentioned adding them to the attendance sheet and then linked the "Hackers arrive" section in the schedule?

**Comment by maxwofford** [comment]
@zachlatta Updated.

**Comment by maxwofford** [comment]
Derp. clicked the wrong button.

File: cohort_4/playbook/activities.md [hunk 1]
```diff
@@ -11,6 +10,13 @@
     - [Feedback Forms](#feedback-forms)
 
 <!-- markdown-toc end -->
+# Activities
+
```
Comment (zachlatta) [inline cohort_4/playbook/activities.md]: Can you remove this extra newline?

File: cohort_4/playbook/activities.md [hunk 2]
```diff
@@ -11,6 +10,13 @@
...
+
+## Hackers arrive
+
+Hacker attendance can be put into
+https://docs.google.com/spreadsheets/d/1Em5tUQdm4dXnthtWr4zY4KR9Q6JZMV_Hl2n6G9Ti9Bg/edit.
```
Comment (zachlatta) [inline cohort_4/playbook/activities.md:18]: Perhaps make the wording a bit more direct?

Ex. "When a hacker arrives, mark their arrival in the attendance over at https://docs.google.com/spreadsheets/d/1Em5tUQdm4dXnthtWr4zY4KR9Q6JZMV_Hl2n6G9Ti9Bg/edit."

File: cohort_4/playbook/schedule.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Schedule
 
+## Attendance
+
+Hacker attendance should be tracked [here](https://docs.google.com/spreadsheets/d/1Em5tUQdm4dXnthtWr4zY4KR9Q6JZMV_Hl2n6G9Ti9Bg/edit)
+
```
Comment (zachlatta) [inline cohort_4/playbook/schedule.md]: Now that we have the "Hackers arrive" section in `activities.md`, do we still need this section?

**Comment by maxwofford** [comment]
@zachlatta Addressed each comment and amended the PR.

**Comment by maxwofford** [comment]
_merge conflict resolved_

**Comment by zachlatta** [comment]
Cherry-picked onto master in 442fbfe341edb727aa5f091941e77d8ef245280d.

**Pull request closed without merging**